packages/dd-trace/src/config.js

```diff
@@ -284,7 +292,7 @@ class Config {
     const DD_TRACE_STATS_COMPUTATION_ENABLED = coalesce(
       options.stats,
       process.env.DD_TRACE_STATS_COMPUTATION_ENABLED,
-      false
+      inGCPFunction()
```

Comment:
mini agent doesnt compute any trace stats, but exposes a trace stats endpoint. We want this var true by default for gcp functions